lib/gitlib.sh

```diff
@@ -23,6 +23,8 @@
 GHCURL="curl -s --fail --retry 10 -u ${GITHUB_TOKEN:-$FLAGS_github_token}:x-oauth-basic"
 JCURL="curl -g -s --fail --retry 10"
 K8S_GITHUB_API='https://api.github.com/repos/kubernetes/kubernetes'
+K8S_GITHUB_RAW='https://raw.githubusercontent.com/kubernetes'
```

Comment:
this one is subtly different, since it's the link to the org, not to the repo (unlike most of the other K8S URLs here). not sure if we want to make that more obvious?